client/db_test.go

```diff
@@ -32,7 +33,9 @@ import (
 
 func setup() (*server.TestServer, *client.DB) {
 	s := server.StartTestServer(nil)
-	db, err := client.Open("https://root@" + s.ServingAddr() + "?certs=test_certs")
+	db, err := client.Open(fmt.Sprintf("https://%s@%s?certs=test_certs",
```

Comment:
mind changing test_certs to `security.EmbeddedCertsDir` while you're touching these?